### PR TITLE
docs: changelog 5.13.1

### PR DESCRIPTION
**Pull request by MadCcc**
- 🐞 Fix Checkbox type error in React@16. [#46962](https://github.com/ant-design/ant-design/pull/46962) [@crazyair](https://github.com/crazyair)

---

- 🐞 修复 Checkbox 组件在 React@16 版本下 Typescript 报错。[#46962](https://github.com/ant-design/ant-design/pull/46962) [@crazyair](https://github.com/crazyair)

[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md?plain=1)]

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Workflow
- [ ] Other (about what?)

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |           |
| 🇨🇳 Chinese |           |

### ☑️ Self-Check before Merge

⚠️ Please check all items below before requesting a reviewing. ⚠️

- [ ] Doc is updated/provided or not needed
- [ ] Demo is updated/provided or not needed
- [ ] TypeScript definition is updated/provided or not needed
- [ ] Changelog is provided or not needed

### CODE REVIEW

**Review by yoyo837 — APPROVED** [review]

File: CHANGELOG.zh-CN.md [hunk 1]
```diff
@@ -16,6 +16,12 @@ tag: vVERSION
 
 ---
 
+## 5.13.1
+
+`2024-01-15`
+
+- 🐞 修复 Checkbox 组件在 React@16 版本下 Typescript 报错。[#46962](https://github.com/ant-design/ant-design/pull/46962) [@crazyair](https://github.com/crazyair)
```
Comment (afc163) [inline CHANGELOG.zh-CN.md:23]:
```suggestion
- 🐞 修复 Checkbox 组件在 `@types/react` 16 或 17 版本下 Typescript 报错。[#46962](https://github.com/ant-design/ant-design/pull/46962) [@crazyair](https://github.com/crazyair)
```

**Review by crazyair — APPROVED** [review]

File: CHANGELOG.en-US.md [hunk 2]
```diff
@@ -16,6 +16,12 @@ tag: vVERSION
 
 ---
 
+## 5.13.1
+
+`2024-01-15`
+
+- 🐞 Fix Checkbox type error in React@16. [#46962](https://github.com/ant-design/ant-design/pull/46962) [@crazyair](https://github.com/crazyair)
```
Comment (MadCcc) [inline CHANGELOG.en-US.md:23]:
```suggestion
- 🐞 Fix Checkbox type error with `@types/react` version 16 or 17. [#46962](https://github.com/ant-design/ant-design/pull/46962) [@crazyair](https://github.com/crazyair)
```

**Comment by li-jia-nan** [comment]
低版本的 @types/react 18 也存在问题

**Comment by yoyo837** [comment]
> 低版本的 @types/react 18 也存在问题

知道最低哪个版本的18是OK的吗？那日志文件可以写 16、17、18@xx以下

**Comment by afc163** [comment]
18 里的低版本就不解决了，没办法兼容每一个子版本的，能升级就让用户升级。

**Comment by crazyair** [comment]
> 18 里的低版本就不解决了，没办法兼容每一个子版本的，能升级就让用户升级。

已经解决了，现在是 changelog 要不要透出 18，我觉得可以不用加。

**Pull request merged**